helm-charts/kiali-server/crds/crds.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
If this is a fork of the regular Kiali chart, what are the differences from upstream?  To upgrade Kiali, would we diff this fork to the new upstream version?

Reply:
It's embarrassing, but I can't for the life of me remember what I did here... All I can remember is that I couldn't get Kiali to work. I did a diff with the original kiali-server Helm chart, and there are a lot of differences, too many in my opinion. I can't remember exactly from where I cloned this kiali-server Helm chart...

Anyway, I guess I will revisit that at some point and clear up everything. For the time being, I would suggest I remove this Helm chart as Kiali is not installed by the `setup-cluster.sh` script anyway. Would you agree with that?

Reply:
Yes, let's remove the Kiali chart.